docs/archive/markup-support.md

```diff
@@ -0,0 +1,167 @@
+# History
+
+To understand the current state of MF2 markup support, the below captures
+moments where markup support developed either key ideas or implementations
+within the repository.
+
+The following events track the "Status" of the markup feature. Status is a
+perception of the group's collective sentiment, as well as a set of traits
+desired from the feature.
+
+The "Takeaway"s entries below are known imperfect. Takeaway bullets are meant to
+represent how collective sentiment or code have tangibly shifted over the
+project lifetime, not to represent the perfect synopsis of an event.
+
+## 27 Nov, 2019
```

Comment:
It's usually a good idea to put the most recent date at the top and "push" dates down. This makes it easier to find the current state later!

Reply:
3690b9b